biscuit/core/layout/base/content/editors/__init__.py

```diff
@@ -45,13 +48,21 @@ def __init__(self, master: ContentPane, *args, **kwargs) -> None:
         self.editorsbar.grid(row=0, column=0, sticky=tk.EW, pady=(0,1))
         self.tabs = self.editorsbar.tabs
 
-        self.editors = []
-        self.closed_editors = {}
+        self.editors: List[Editor] = []
+        self.closed_editors: List[Editor] = {}
+
         self.empty = True
         self.emptytab = Empty(self)
         self.emptytab.grid(column=0, row=1, sticky=tk.NSEW)
 
-        self.default_editors = [Welcome(self)]
+        self.default_editors: List[Editor] = [Welcome(self)]
+        self.actionset = ActionSet("Switch to active editors", "active:", [])
+        self.base.palette.register_actionset(self.get_active_actionset)
+    
+    def get_active_actionset(self) -> ActionSet:
+        "Generates the active editors actionset"
+        self.actionset.update([(editor.filename, editor) for editor in self.editors])
+        return self.actionset
```

Comment:
this can make the palette search a bit slow if there are many active items, but fine